README.md

```diff
@@ -32,14 +32,14 @@ Git clone this repository then:
 
     npm start
 
-#### Test run via local command line
-
-    SLACK_TOKEN=secret GH_TOKEN=secret GH_REPOS=johndoe/somerepo,johndoe/anotherrepo ./bin/pr-police --cli
-
 This will start the server locally until `Ctrl-C` is pressed.
 
 **Note:** You will need to pass all the required env vars.
 
+#### Test run via local command line
+
+    SLACK_TOKEN=secret GH_TOKEN=secret GH_REPOS=johndoe/somerepo,johndoe/anotherrepo ./bin/pr-police --cli
+
```

Comment:
This change is confusing to me... `npm start` is only intended for use on heroku, right? You can't really press `Ctrl-C` on heroku can you, so wouldn't that only be relevant for CLI?

Reply:
You can run `npm start` locally and test out changes you're making to the server by typing commands into slack.

Reply:
Ah, I see. Maybe it's worth mentioning that here, too.  For example:

Run locally with selection of environment variables to post to Slack:
`SLACK_TOKEN=xxx GH_TOKEN=yyy GH_REPOS=test/repo SLACK_CHANNELS=pr-police-test npm start`

Or run with the cli option to send output to your console.